.gitmodules

```diff
@@ -29,4 +29,7 @@
 [submodule "src/third_party/seastar"]
 	path = src/third_party/seastar
 	url = https://github.com/scylladb/seastar.git
-	ignore = dirty
\ No newline at end of file
+	ignore = dirty
+[submodule "src/third_party/fmt"]
```

Comment:
I noticed that `gflags & glog` now also offer CMakeLists.txt upstream too. So could be nice to be `as much submodule as possible`

But not a requirement for this PR.

Reply:
sounds good to me, but why use submodule dependencies if there is good system-level support?

Reply:
makes sense. 

Can we add a quick section to `contributing.md` thtat says basically your approach which i like. 

`We either use apt-get/dnf-install or add_subdirectory for 3rd party deps`

Reply:
cool. yeh i think this is a very common approach. as an extreme example, you'd be hard pressed to find a project that did `add_subdirectory(binutils)` or whatever.